src/backend/vulkan/src/physical_device.rs

```diff
@@ -582,6 +583,18 @@ impl PhysicalDeviceInfo {
             requested_extensions.push(vk::KhrGetDisplayProperties2Fn::name()); // TODO NOT NEEDED, RIGHT?
         }
 
+        if requested_features.contains(Features::EXTERNAL_MEMORY) {
```

Comment:
need to also check for `self.api_version`. Some of these extensions were promoted into core at some point, which means they may no longer be advertised.

Reply:
I didn't know that. I have fixed the external memory extensions and adjusted functions accordingly.